pkg/object/meshcontroller/worker/api.go

```diff
@@ -31,19 +31,19 @@ const (
 	meshNacosPrefix = "/nacos/v1"
 )
 
-func (wrk *Worker) runAPIServer() {
+func (worker *Worker) runAPIServer() {
```

Comment:
proposal `w`. [1]

[1] https://github.com/golang/go/wiki/CodeReviewComments#receiver-names

Reply:
In some files, it's conflicted with `http.ResponseWriter `'s `w.`,e.g., 
```
func (worker *Worker) nacosRegister(w http.ResponseWriter, r *http.Request)
``` 
But I can replace the `woker` with `w`  in other not-conflicted files.

Reply:
I got it. then `worker` it is, for consistency.